icons/aliases.json

```diff
@@ -250,30 +250,20 @@
     "address": "0x881b06da56BB5675c54E4Ed311c21E54C5025298"
   },
   {
-    "name": "LINK",
+    "name": "ChainLink Token",
     "symbol": "LINK",
-    "address": "0x514910771af9ca656af840dff83e8264ecf986ca"
+    "address": "0x514910771AF9Ca656af840dff83E8264EcF986CA"
   },
   {
     "name": "HEGIC yVault",
     "symbol": "yvHEGIC",
-    "address": "0x19db27D2E9E4B780CF9A296D575aBbddEe1578DA"
+    "address": "0xe11ba472F74869176652C35D30dB89854b5ae84D"
   },
   {
-    "name": "HEGIC",
+    "name": "Hegic",
     "symbol": "HEGIC",
     "address": "0x584bC13c7D411c00c01A62e8019472dE68768430"
   },
-  {
```

Comment:
Can you put this back?

Reply:
Sure, it's not a live vault but I can add it